### ISSUE TITLE
Use CaptureHistory() instead of HistoryCarbonCopy

### ISSUE THREAD

**Issue by Garados007** [byte-physics/igortest#460]
We can use `CaptureHistoryStart()` and `CaptureHistory()` to capture all history output between the start of the capture and the current invocation. Igor can maintain multiple history captures at the same time and all we have to do is to remember the reference number returned by `CatuptureHistoryStart()` and provide it to `CaptureHistory()`. We can call `CaptureHistory()` multiple times and can stop a specific capture when there is no longer a need for it.

```igorpro
Function test()
	variable refNum = CaptureHistoryStart()
	print "Hello world!"
	
	string capture = CaptureHistory(refNum, 0) // 0 is to keep capture active
	print "🌲"
	print capture // should be "Hello world!"
	
	capture = CaptureHistory(refNum, 1) // 1 stops capture
	print capture // should be "Hello world!\n🌲\nHello world!"
End
```

We currently use a single notebook with the specific name `HistoryCarbonCopy` which contains magically all history output including the one there is no use for us (because its the output before the current test suite, test case, ...). We capture at the start and end of all test runs/suites/cases all historic output and create the desired string slices at the end of them.

I think we can improve our data collection when we use `CaptureHistoryStart()` and `CaptureHistory()` instead and do not rely on hidden magical notebooks.

**Comment by t-b** [comment]
Good idea, but right now IP6 does not support fancy things like `CaptureHistoryStart`/`CaptureHistory`.

**Comment by Garados007** [comment]
> Good idea, but right now IP6 does not support fancy things like `CaptureHistoryStart`/`CaptureHistory`.

That's a pity. There was nothing in the Igor documentation about when it was introduced. Maybe keep this in mind and come back when we decide to drop support for Igor 6.

**Comment by Garados007** [comment]
Do we want to add this to [Version 2.0](https://github.com/byte-physics/igortest/milestone/6)?

**Comment by Garados007** [comment]
Hmm. I started Igor 6 locally and I run the test code above. Apart from the unicode tree everything was fine:

![grafik](https://github.com/byte-physics/igortest/assets/12125053/2233d190-ea71-4992-a2be-20cdfa1bb28c)

And its also in the Igor 6 documentation.

**Comment by t-b** [comment]
Interesting, you are right it is present in IP6. So let's switch to that!
